### PR TITLE
Add the support for -webkit-text-security in Release 114

### PR DESCRIPTION
**Pull request by andreydanil**
Firefox is the only widely used engine not supporting -webkit-text-security, and it seems to be in use in the wild to conceal password fields, as per this [webcompat.com report](https://github.com/webcompat/web-bugs/issues/114591). Chrome Platform Status [implies that its usage](https://chromestatus.com/metrics/css/timeline/popularity/322) is low (~0.3%), but slowly growing.

### Description

https://github.com/whatwg/compat/issues/240

### Motivation

Fixed in https://bugzilla.mozilla.org/show_bug.cgi?id=1826629 and released in Firefox 114.

### Additional details

* https://hg.mozilla.org/mozilla-central/rev/c5031904ac17
* https://hg.mozilla.org/mozilla-central/rev/0b42da869302

### CODE REVIEW

**Comment by andreydanil** [comment]
PR for the browser compatibility update: https://github.com/mdn/browser-compat-data/pull/19887.

File: files/en-us/web/css/-webkit-text-security/index.md [hunk 1]
```diff
@@ -2,14 +2,11 @@
...
 page-type: css-property
-status:
-  - non-standard
 browser-compat: css.properties.-webkit-text-security
 ---
 
-{{CSSRef}} {{ Non-standard_header() }}
-
 **`-webkit-text-security`** is a non-standard CSS property that obfuscates characters in a {{HtmlElement("form")}} field (such as {{HtmlElement("input")}} or {{HtmlElement("textarea")}}) by replacing them with a shape. It only affects fields that are _not_ of `type=password`.
+**`-webkit-text-security`** is supported as of [Firefox 114](https://bugzilla.mozilla.org/show_bug.cgi?id=1826629).
```
Comment (dipikabh) [inline files/en-us/web/css/-webkit-text-security/index.md:9]: This type of information typically goes into the release notes docs: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/114#css.

Source: https://github.com/mdn/content/blob/main/files/en-us/mozilla/firefox/releases/114/index.md

Also, to refer to the bug in the release notes doc, we use the format  `[Firefox bug <number>](https://bugzil.la/<number>)`.

```suggestion

```

File: files/en-us/web/css/-webkit-text-security/index.md [hunk 2]
```diff
@@ -2,14 +2,11 @@
 title: "-webkit-text-security"
 slug: Web/CSS/-webkit-text-security
 page-type: css-property
-status:
-  - non-standard
 browser-compat: css.properties.-webkit-text-security
 ---
 
-{{CSSRef}} {{ Non-standard_header() }}
```
Comment (dipikabh) [inline files/en-us/web/css/-webkit-text-security/index.md:10]: We do need to keep the `{{CSSRef}}` macro because it generates the CSS sidebar.

**Review by dipikabh — CHANGES_REQUESTED** [review]
Hello @andreydanil, thanks for opening the PRs to update content and BCD!

We add the Firefox release specific changes to our release docs. Would it be possible for you to update that file as part of this PR (basically move the update info from `webkit-text-security` to `releases/114`? Thanks!

File: files/en-us/web/css/-webkit-text-security/index.md (hunk 1 above)
Comment (andreydanil) [reply files/en-us/web/css/-webkit-text-security/index.md]: Thanks for the feedback. I've updated the PR. Feel free to update the verbiage.

Could you also please review this PR: https://github.com/mdn/browser-compat-data/pull/19887 to mark that the `-webkit-text-security` CSS property is supported in FIrefox 114?

File: files/en-us/mozilla/firefox/releases/114/index.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ This article provides information about the changes in Firefox 114 that affect d
 #### Removals
 
 ### CSS
+- The [-webkit-text-security](/en-US/Web/CSS/-webkit-text-security) support has been successfully addressed in [Firefox bug 1826629](https://bugzil.la/1826629). Users can now utilize the `-webkit-text-security` CSS property to control the rendering of text security features.
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/114/index.md:20]:
```suggestion

- The [-webkit-text-security](/en-US/Web/CSS/-webkit-text-security) property that lets you replace characters with shapes is now supported. You can now control the rendering of text security using this property. ([Firefox bug 1826629](https://bugzil.la/1826629)). 

```

**Review by dipikabh — CHANGES_REQUESTED** [review]
Thanks a lot for the updates!

I've suggested a rewording here. Please cross-check for technical accuracy.

- On the content page, two things are still required:
  - In the first sentence, "`webkit-text-security` is a non-standard CSS property" should now be "`webkit-text-security` is a CSS property"
  - `{{ Non-standard_header() }}` needs to be removed
- I left a comment in the BCD PR.

File: files/en-us/mozilla/firefox/releases/114/index.md (hunk 3 above)
Comment (andreydanil) [reply files/en-us/mozilla/firefox/releases/114/index.md:20]: Thanks! I will make those updates shortly.

File: files/en-us/mozilla/firefox/releases/114/index.md [hunk 4]
```diff
@@ -17,6 +17,8 @@ This article provides information about the changes in Firefox 114 that affect d
 
 ### CSS
 
+- The [-webkit-text-security](/en-US/Web/CSS/-webkit-text-security) property that lets you replace characters with shapes is now supported. You can now control the rendering of text security using this property. ([Firefox bug 1826629](https://bugzil.la/1826629)).
```
Comment (dipikabh) [inline files/en-us/mozilla/firefox/releases/114/index.md:20]:
```suggestion
- The [-webkit-text-security](/en-US/Web/CSS/-webkit-text-security) property that lets you replace characters with shapes is now supported. You can now control the rendering of text security using this property. ([Firefox bug 1826629](https://bugzil.la/1826629)).
```

**Review by dipikabh — APPROVED** [review]
Thanks a lot! 👍

**Pull request merged**